doc/odoc_for_authors.mld

```diff
@@ -508,15 +510,15 @@ the signature.
 These tags have a single line of data associated with them, [string] in the
 examples below. They are:
 
-- [@author string] - allows the author of the element to be specified.
-- [@since string] - declares from which version the element has been available.
-- [@version string] - declares the version of the element itself.
-- [@canonical string] - declares the path to the canonical instance of this element.
-Can be applied to modules, module types and types. See the {{!page-features.canonical}
-Language Features} page for more details.
+- [@author string] - allows the element's author to be specified
+- [@since string] - declares from which version the element has been available
+- [@version string] - declares the version of the element itself
+- [@canonical string] - declares the path to this element's canonical instance.
+It can be applied to modules, module types, and types. See the {{!page-features.canonical}
+Language Features} page for more details
 
-The content of the tag is then the rest of the line, and it is uninterpreted,
-i.e., there shouldn't be any odoc markup.
+The tag's content is then the rest of the line. It is also uninterpreted,
+i.e., there shouldn't be any [odoc] markup.
```

Comment:
```suggestion
The tag's content fills the rest of the line. It is also uninterpreted,
i.e., there shouldn't be any [odoc] markup.
```
Is this still accurate?

Reply:
It talks about how it interprets the tags in the source code so it's more like "The tag's content is read from the rest of the line."